Examples/create-app.js

```diff
@@ -90,6 +90,9 @@ function createCodePushApp(name, os) {
         execCommand(`appcenter codepush deployment add -a ${owner}/${name} Staging`);
     } catch (e) {
         console.log(`App "${name}" already exists \n`);
+        const ownerResult = execCommand(`appcenter profile list --output json`);
```

Comment:
nit: If we don't insert any expression into the string, could you replace it to double quotes?